quickjs.h

```diff
@@ -38,13 +38,12 @@ extern "C" {
 
 #if defined(__GNUC__) || defined(__clang__)
 #define js_force_inline       inline __attribute__((always_inline))
-#define __js_printf_like(f, a)   __attribute__((format(printf, f, a)))
 #define JS_EXTERN __attribute__((visibility("default")))
 #else
 #define js_force_inline  inline
-#define __js_printf_like(a, b)
 #define JS_EXTERN /* nothing */
 #endif
+#define __JS_FORMAT_PRINTF __attribute__((format(printf, 2, 3)))
```

Comment:
I was wondering why is there underscore in this particular case

Reply:
Leftover from the first approach, I'll remove it. Thanks for taking a look!